toolchain/check/handle_binding_pattern.cpp

```diff
@@ -19,15 +19,19 @@
 
 namespace Carbon::Check {
 
+auto HandleParseNode(Context& context, Parse::UnderscoreNameId node_id)
+    -> bool {
+  context.node_stack().Push(node_id, SemIR::NameId::Underscore);
+  return true;
+}
+
 static auto HandleAnyBindingPattern(Context& context, Parse::NodeId node_id,
```

Comment:
It looks like `HandleAnyBindingPattern` is over 300 lines; maybe worth a TODO to factor things out a little?